pod/perlgit.pod

```diff
@@ -785,6 +785,23 @@ When describing the merge commit, explain the purpose of the branch, and
 keep in mind that this description will probably be used by the
 eventual release engineer when reviewing the next perldelta document.
 
+If the subsequent I<push> fails then you must be careful on how you I<rebase>.
+If you use
+
+  % git rebase p5p/blead
+
+or
+
+  % git pull --rebase
+
+then your carefully created merge commit will be lost! To avoid this you
+can use:
+
+  % git fetch p5p
+  % git rebase --preserve-merges p5p/blead
```

Comment:
--preserve-merges was removed in git 2.34, --rebase-merges should be used instead.

https://github.com/git/git/blob/master/Documentation/RelNotes/2.34.0.txt